src/app/core/header/user/user.component.test.ts

```diff
@@ -0,0 +1,31 @@
+import { UserComponent } from './user.component';
+import { LOGIN_ROUTE, REGISTRATION_ROUTE } from '../header-navigation-routes';
+
+describe('User', () => {
+    let sut;
+    let profileService;
+
+    beforeEach(() => {
+        profileService = {
+            getUserName: jasmine.createSpy('getUserName')
+        };
+        sut = new UserComponent(profileService);
+    });
+
+    describe('routing', () => {
+        it('should show log in route', () => {
+            expect(sut.loginRoute).toEqual(LOGIN_ROUTE);
```

Comment:
In tests will be better to use magic strings instead of importing simple variables.